### PR TITLE
update InterfacesFor to use GroupVersion

### PR DESCRIPTION
**Pull request by deads2k**
update InterfacesFor to use GroupVersion.  Should be non-contentious. 

@caesarxuchao @liggitt

### CODE REVIEW

File: pkg/api/latest/latest.go [hunk 1]
```diff
@@ -144,5 +144,5 @@ type GroupMeta struct {
 
 	// InterfacesFor returns the default Codec and ResourceVersioner for a given version
 	// string, or an error if the version is not known.
```
Comment (liggitt) [inline pkg/api/latest/latest.go]: Update godoc

**Comment by liggitt** [comment]
nit on godoc, LGTM otherwise

**Comment by deads2k** [comment]
godoc updated.

**Comment by deads2k** [comment]
@k8s-bot test this please
@k8s-bot unit test this please

**Comment by deads2k** [comment]
@k8s-bot unit test this

**Pull request merged**